NEWS.md

```diff
@@ -6,6 +6,17 @@
 
 * Ignore `.env`, `.venv`, and `venv` files only when they reference Python
   virtual environments. (#972)
+  
+* Add `envManagement`, `envManagementR` and `envManagementPy` args when writing
+  the `manifest.json`. These args specify whether Posit Connect should install
```

Comment:
nit: This applies to `deployApp()` and `writeManifest()`. Most NEWS is in terms of the modified functions.

> * `deployApp()` and `writeManifest()` accept optional `envManagement`, `envManagementR`, and `envManagementPy` arguments. These arguments indicate whether or not Posit Connect should install and manage package environments for the content. ...